tools/disasm/variables.txt

```diff
@@ -4729,7 +4729,7 @@
     0x8082B604:("sMaskEquipMagicArrowSlotHoldTimer","UNK_TYPE2","",0x2),
     0x8082B608:("sMaskEquipAnimTimer","UNK_TYPE2","",0x2),
     0x8082B60C:("gMaskPlayerFormSlotRestrictions","UNK_TYPE1","",0x1),
-    0x8082B684:("sMoonMasksTakenBits","UNK_TYPE2","",0x2),
+    0x8082B684:("sMasksGivenOnMoonBits","UNK_TYPE2","",0x2),
```

Comment:
Think this didn't get updated?

Reply:
This seems to line up with the current iteration of this variable name